src/test/java/com/exasol/adapter/dynamodb/DynamodbAdapterIT.java

```diff
@@ -59,6 +61,7 @@ static void beforeAll() throws DynamodbTestInterface.NoNetworkFoundException, SQ
         exasolTestDatabaseBuilder.uploadMapping(MappingTestFiles.DATA_TYPE_TEST_MAPPING_FILE_NAME);
         exasolTestDatabaseBuilder.uploadMapping(MappingTestFiles.DOUBLE_NESTED_TO_TABLE_MAPPING_FILE_NAME);
         exasolTestDatabaseBuilder.uploadMapping(MappingTestFiles.TO_JSON_MAPPING_FILE_NAME);
+        Thread.sleep(3000);
```

Comment:
It's better to add a comment why you need it here. People who read your code would probably like to know it